spec/beaker/hypervisor/vsphere_spec.rb

```diff
@@ -58,6 +58,20 @@ module Beaker
 
       end
 
+      it 'does not provisions hosts if provision is set to false' do
+        MockVsphereHelper.powerOff
+        hosts = make_hosts()
+        hosts[0]["provision"] = false
+        vsphere = Beaker::Vsphere.new( hosts, make_opts )
+
+        vsphere.provision
+
+        hosts.each do |host|
+          expect( MockVsphereHelper.find_vm( host.name ).powerState ) == "poweredOff"
```

Comment:
@kevpl Your comment is entirely true. That also means that 3 existing vsphere tests are useless at this time...

I tried to write a good test but I can't get it to work. This is the closest I could get.
```
it 'does not resets to the given snapshot if provision is set to false' do
  MockVsphereHelper.powerOff
  hosts = make_hosts()
  hosts[0]["provision"] = false
  vsphere = Beaker::Vsphere.new( hosts, make_opts )
  expect( MockVsphereHelper ).to receive( :find_snapshot ).exactly( hosts.length - 1 ).times
  vsphere.provision
end
```
Rather dull, but it seems that the effect of this feature can already be achieved by simply not providing a snapshot... my bad :|